components/brave_new_tab_ui/components/default/braveToday/customize/Context.tsx

```diff
@@ -29,6 +30,10 @@ interface BraveNewsContext {
   // Publishers to suggest to the user.
   suggestedPublisherIds: string[]
   updateSuggestedPublisherIds: () => void
+  optInPrefEnabled: boolean | undefined
```

Comment:
I don't know how I feel about exposing these in multiple places, rather than just going through the prefs hook.

Maybe instead of just reexposing the `useNewTabPref` stuff we could have a function which does both setOptIn & setShow?

const setShowBraveNews(optIn: boolean, show: boolean) => { ... }

That way we couldn't forget to do one of them (which was the cause of @simonhong's issue the other day).

Reply:
If not, I think I'd prefer users of the pref to just go straight through the `useNewTabPref` rather than drilling it through the Context here.